core/src/main/java/org/apache/iceberg/BaseFilesTable.java

```diff
@@ -108,7 +108,8 @@ public TableScan appendsAfter(long fromSnapshotId) {
     }
 
     /**
-     * @return list of manifest files to explore for this files metadata table scan
+     * Returns list of {@link ManifestFile} files to explore for this file's metadata table scan
+     * @return list of manifest files
```

Comment:
Why is this change needed?

Reply:
the static check warning log ，so I think should need change:

```
[MissingSummary] A summary fragment is required; consider using the value of the @return block as a summary fragment instead.
     * @return list of manifest files to explore for this files metadata table scan
       ^
    (see https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment)
  Did you mean '*Returns list of manifest files to explore for this files metadata table scan.'?
```